_data/projects/npjs-basic.yml

```
@@ -0,0 +1,10 @@
+name: npjs-basic
+desc: A numpy-like library for npm
+site: https://example.org/your-project-link-here
+tags:
+  - javascript
+  - node.js
+  - npm
+upforgrabs:
+  name: good first issue, up-for-grabs
+  link: https://github.com/hrishibawane/npjs-basic
```

This then becomes the link that corresponds with that label:

```suggestion
  link: https://github.com/hrishibawane/npjs-basic/labels/up-for-grabs
```